src/main/kotlin/earth/terrarium/techarium/common/util/Extensions.kt

```diff
@@ -0,0 +1,13 @@
+package earth.terrarium.techarium.common.util
```

Comment:
This should be in a package called `utils` as plural is used for all other packages.
This should also not be called `Extensions` something like BlockUtils would be better as we should not be putting all extensions in one file.